### PR TITLE
Fixed docs formatting problems related to relref

### PR DESCRIPTION
**Pull request by chrisnegus**
*Description of changes:*
An extra line break is being added to end of links marked with "relref" inside of Note and Warning alerts. I'll look into the underlying problem later. But in the meantime, I've done various workarounds to get the text to display properly. View the rendered pages [here](http://ec2-3-142-142-41.us-east-2.compute.amazonaws.com:1313/docs/).

### CODE REVIEW

**Comment by rothgar** [comment]
haha, if it works it works. Do you have a screenshot?

**Comment by chrisnegus** [comment]
> haha, if it works it works. Do you have a screenshot?

Yeah. I'll add those. I forgot to fetch the latest main, so I'll have to rebase. Then I'll add screen shots.

**Comment by ivyostosh** [comment]
Overall looks good to me. I also checked out a few locally and it seems be to doing its job. Thanks for taking care of it!

I can add a new label once you rebase as well.

/approve
/lgtm

File: docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md [hunk 1]
```diff
@@ -58,6 +58,14 @@ Follow these steps to create an EKS Anywhere cluster.
 
    The Amazon EKS Anywhere Curated Packages are only available to customers with the Amazon EKS Anywhere Enterprise Subscription. To request a free trial, talk to your Amazon representative or connect with one [here](https://aws.amazon.com/contact-us/sales-support-eks/). Cluster creation will succeed if authentication is not set up, but some warnings may be genered.  Detailed package configurations can be found [here]({{< relref "../../tasks/packages" >}}).
 
+     {{% alert title="Note" color="primary" %}}
```
Comment (TerryHowe) [inline docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md:61]: Why is this coming back? This is not correct if it has to come back.

**Review by TerryHowe — CHANGES_REQUESTED** [review]

File: docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md (hunk 1 above)
Comment (chrisnegus) [reply docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md:61]: My mistake. I was working from an old branch. You mean I should remove the whole note, right? I went ahead and did that.

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

**Pull request merged**